### PR TITLE
Contentstack enhancement

### PR DESCRIPTION
**Pull request by joe-ayoub-segment**
Adding Hybrid Action for Contentstack

Updates the integration object with a boolean value. 
true - there are new traits
false - there are no new traits

The reason is to reduce the number of trait creation requests being sent from the associated Cloud Mode Actions. 

## Testing

Testing not needed yet - no customers using the Integraiton

Note: I've left some console.log calls in the code to help verify this in Production. Will raise a new PR to remove them.

### CODE REVIEW

File: packages/browser-destinations/destinations/contentstack-browser-plugins/src/contentstackPlugin/index.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      label: 'User traits',
+      description: 'User Profile traits to send to Contentstack',
+      required: true
+    }
+  },
+  lifecycleHook: 'enrichment',
+  perform: (_, { context, analytics, payload }) => {
+    const storage = (analytics.storage as UniversalStorage<Record<string, string>>) ?? storageFallback
+
+    const { traits } = payload
```
Comment (pooyaj) [inline packages/browser-destinations/destinations/contentstack-browser-plugins/src/contentstackPlugin/index.ts:28]: should we do an `undefined/null` check for traits and short-circuit the method in that case?

**Review by pooyaj — COMMENTED** [review]
Reviewed the browser plugin. Looks great. Small comment + please remove the few `console.logs` across the code. Otherwise looks good to me

cc - @joe-ayoub-segment

File: packages/browser-destinations/destinations/contentstack-browser-plugins/src/contentstackPlugin/index.ts (hunk 1 above)
Comment (joe-ayoub-segment) [reply packages/browser-destinations/destinations/contentstack-browser-plugins/src/contentstackPlugin/index.ts:28]: done

**Comment by joe-ayoub-segment** [comment]
> Reviewed the browser plugin. Looks great. Small comment + please remove the few `console.logs` across the code. Otherwise looks good to me
> 
> cc - @joe-ayoub-segment

I'd like to keep the comments so I can test post deploy. Will remove them in a subsequent PR.

**Review by sayan-das-in — APPROVED** [review]

**Comment by joe-ayoub-segment** [comment]
server side code deployed. Client side code needed.

**Pull request merged**